src/lib/shared/input.ts

```diff
@@ -374,4 +392,12 @@ export const inputPhrases: InputPhrases = {
   "input.valueAddButton": "Add value",
   "input.valueLabel": "Value",
   "input.valuePlaceholder": "Add a value",
+  "input.writeOnly.alert":
+    "The value entered in this field will no longer be viewable once you click connect or save this page. However, you will still be able to enter a new value at any time.",
+  "input.writeOnly.replaceValueTooltip": "Replace Existing Value",
+  "input.writeOnly.replaceValueConfirmation.message":
+    'The value entered in the field below will no longer be viewable once you click "Save Value."',
```

Comment:
Super nit....and super optional... Could put the `.` outside of `"Save Value"`

Reply:
Updated